.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,24 +1,26 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
-title: ''
-labels: ''
-assignees: ''
-
+name: Feature Request
+about: Suggest a new feature or enhancement for this project
+title: ""
+labels: "Status: Needs Review"
+assignees: ""
 ---
 
-#### Description
-<!-- Brief description of why this feature would be useful. What problems would it solve and what are the use cases? -->
+<!-- Thank you for taking the time to submit a new feature request to The Odin Project. In order to get issues closed in a reasonable amount of time, you must include a baseline of information about the feature/enhancement you are proposing. Please read this template in its entirety before filling it out to ensure that it is filled out correctly. -->
+
+<!-- Complete the following REQUIRED checkboxes by replacing the whitespace between the square brackets with an 'x', e.g. [x]. -->
+
+-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
+-   [ ] The title of this issue follows the `where the change occurs: brief description of request` format, e.g. `Global SCSS file: Buttons should have XYZ`
+
+<!-- The following checkbox is OPTIONAL. Completing it does not guarantee you will be assigned this issue, but rather lets us know you are interested in working on it. -->
+
+-   [ ] I would like to be assigned this issue to work on it
+
+**1. Description of the Feature Request**
 
-#### Acceptance Criteria:
-<!-- Description of how this feature should work or the steps that need to be taken by the user to meet the acceptance criteria:
+<!-- A clear and concise description of what the feature is. Also include how it would be useful/beneficial or what problem(s) it would solve. -->
 
-For example:
-1. Log in
```

Comment:
Same here please

Reply:
So I'm not exactly sure if this examples really makes sense in this section as proposed, just because it's more "here's why I think this feature would be a good idea". That said, I could just add back in the Acceptance Criteria section and keep the example (or a similar example) there. Let me know what you think.

Reply:
Yeah, I think I removed it to make things more "general", but thinking about it again I think this section can really work across any repo (and users can just skip it if they don't have any criteria, while maintainers can edit in any criteria as well)